index.js

```diff
@@ -521,6 +544,11 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this.valueOf() <= other.valueOf();
+  }
+
```

Comment:
I believe two values should be considered equal if and only if they can be used interchangeably. `false` and `new Boolean(false)` are not interchangeable, therefore they should not be considered equal. There are many situations in which these values are not interchangeable, but this is probably the most striking:

```javascript
> false ? '😵' : '🙂'
'🙂'

> new Boolean(false) ? '😵' : '🙂'
'😵'
```